core/api/src/graphql/public/types/object/user.ts

```diff
@@ -126,6 +128,26 @@ const GraphQLUser = GT.Object<User, GraphQLPublicContextAuth>({
       },
     },
 
+    scopes: {
+      type: GT.NonNullList(Scope),
+      description:
+        "Retrieve the list of scopes permitted for the user's token or API key",
+      resolve: async (_source, _args, { domainAccount, scope }) => {
+        const isActive = domainAccount.status === AccountStatus.Active
```

Comment:
This code contains domain logic. Better to extract it into domain helper that we can unit test.

Reply:
Updated https://github.com/GaloyMoney/galoy/pull/4100/commits/11c8b0bc6091b86a6f3a6eccd1dbbf3acdfb7c98